docs/development/privacy.md

```diff
@@ -0,0 +1,49 @@
+# Privacy Checklist for ILIAS Development
+
+Start with a quick self-check. Before diving into specific questions, take a moment to reflect on your approach:
+
+- [ ] Have I considered all data protection-relevant aspects of this feature?
+- [ ] Could I clearly explain my solution to a data protection officer?
+- [ ] Would I feel comfortable using this feature on a platform that processes my own data?
+
+If you can confidently answer “yes” to all of the above – great! If not, the checklist below will help identify areas where your implementation can become more privacy-friendly.
+
+### 1. Data Collection
+- [ ] Are personal data being processed? If so, which data exactly?
+- [ ] Is the processing technically necessary – or just “nice to have”?
+- [ ] Is there a way to reduce data usage (smaller scope, shorter storage duration)?
+
+### 2. Defaults & Control
+- [ ] Is the most privacy-friendly setting used by default (Privacy by Default)?
+- [ ] Can admins and, if applicable, users adjust the settings?
+- [ ] Are there tooltips or notices that explain data usage (transparency)?
+
+### 3. Access & Roles
+- [ ] Is access to personal data role-based and restricted?
+- [ ] Is there protection against unintended disclosure (e.g. masking in logs)?
+- [ ] Are sensitive data only displayed or stored when necessary?
+
+### 4. Deletion & Storage Limitation
+- [ ] Can the data be fully deleted if required?
+- [ ] When a user or course is deleted, are all associated data reliably removed?
+- [ ] Is there a feature for automatic or scheduled deletion?
+
+### 5. Export & Access Requests
+- [ ] Can personal data be exported upon request?
```

Comment:
Please clarify what "upon request" means. Please consider posing an open ended question to prompt a more info-rich answer: How can personal data be exported?".